src/Examples/Flowsheets/hda_reaction.py

```diff
@@ -71,31 +72,36 @@ def build(self):
                                             ("R1", "Vap", "toluene"): -1,
                                             ("R1", "Vap", "hydrogen"): -1,
                                             ("R1", "Vap", "methane"): 1,
+                                            ("R1", "Vap", "diphenyl"): 0,
                                             ("R1", "Liq", "benzene"): 0,
                                             ("R1", "Liq", "toluene"): 0,
                                             ("R1", "Liq", "hydrogen"): 0,
-                                            ("R1", "Liq", "methane"): 0}
+                                            ("R1", "Liq", "methane"): 0,
+                                            ("R1", "Liq", "diphenyl"): 0}
```

Comment:
If you are adding diphenyl here, doesn't there need to be a second reaction (otherwise how will it be formed)?